src/components/Messaging/MessageBody.js

```diff
@@ -488,9 +497,41 @@ const MessageBody = ({ messages, messageRecipient, avatar }) => {
         const res = await postMessageSend(messagePayload);
         if (res.error) {
           setErrorMessage(res.message);
-          handleModalOpen();
         } else {
-          history.go(0);
+          const newMessage = {
+            parent_message_id: lastMessage.id ? lastMessage.id : null,
```

Comment:
Couldn't you just simplify it to `lastMessage.id`?

Reply:
referenced to new issue so we can merge.